x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/log_rate_analysis_results_table.tsx

```diff
@@ -123,6 +129,7 @@ export const LogRateAnalysisResultsTable: FC<LogRateAnalysisResultsTableProps> =
     const itemCount = significantItems?.length ?? 0;
 
     let items: SignificantItem[] = significantItems ?? [];
+    // console.log('--- SORT FIELD ---', sortField.props.children[0]);
```

Comment:
Can this be removed?

Reply:
Removed in 7f0c33b865302483af24b4b327c42bfa1d861cae